app/queries/developer_query.rb

```diff
@@ -90,9 +96,22 @@ def empty_search?
       role_levels.empty? &&
       search_query.blank? &&
       countries.blank? &&
+      badges.blank? &&
       !include_not_interested
   end
 
+  def badges_filter_records
+    badges.map { |badge| send(badge) } if badges.any?
```

Comment:
`#send` feels a little dangerous here. Especially because it will be coming directly from user input.

Can we modify this to either a) not use `#send` or b) filter the input to guarantee it is either `recently_active` or `source_contributor` and nothing else?

Reply:
You are correct, given the second thought it is not good idea to use send for such input. Will modify to restrict to only two options